docs/guides/webview-versions.md

```diff
@@ -0,0 +1,186 @@
+[609.4.1.1.1]: https://github.com/WebKit/WebKit/blob/8df64286794c38efa4697b7c24658cb85204a070/Source/WebKit/Configurations/Version.xcconfig
+[609.4.1]: https://github.com/WebKit/WebKit/blob/cb927e6151b5ef49c9ccfb13018f51471f8f1035/Source/WebKit/Configurations/Version.xcconfig
+[609.2.9.1.2]: https://github.com/WebKit/WebKit/blob/ca54d252f3416c3ec64f80a084cb5c4ff7ba24f1/Source/WebKit/Configurations/Version.xcconfig
+[609.1.20.111.8]: https://github.com/WebKit/WebKit/blob/5c90480a38a86464b6b421c2fd28c744b43a4faa/Source/WebKit/Configurations/Version.xcconfig
+[613.3.9.1.5]: https://github.com/WebKit/WebKit/blob/7f88b99524540e94abcdef4d45c1c0324d63fb56/Source/WebKit/Configurations/Version.xcconfig
+
+### Linux (WebKitGTK)
+
+The fractured nature of the Linux ecosystem means it is very hard to compile accurate information about WebKitGTK on the various distros. The table below is a very incomplete list of the most commonly used distributions and their webkit versions. You should always check your distros repositories for up-to-date information.
+
+| Distro                                                         | webkitgtk | WebKit    | Safari Equivalent |
+| -------------------------------------------------------------------------- | --------- | --------- | ------------- |
+| Ubuntu 18.04                                                                                        | 2.20      | [606.1.4] | 12.0              |
+| Debian 9 (with backport), Debian 10                                                      | 2.24      | [608.1.6] | 13.0              |
+| Ubuntu 20.04                                                                                        | 2.28      | [610.1.1] | 14.0              |
+| Debian 11, Ubuntu 18.04 (with update), centos 8 (non-stream)             | 2.32      | [612.1.6] | 15.0              |
+| Debian 10 (with update)                                                                        | 2.34      | [613.1.1] | 15.4              |
+| Debian 11 (with update), Ubuntu 20.04 (with update), Ubuntu 22.04    | 2.36      | [614.1.6] | TP 140 (16.0)     |
+
+[605.1.3]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.18/Source/WebKit/Configurations/Version.xcconfig
+[606.1.4]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.20/Source/WebKit/Configurations/Version.xcconfig
+[610.1.1]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.28/Source/WebKit/Configurations/Version.xcconfig
+[614.1.6]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.36/Source/WebKit/Configurations/Version.xcconfig
+[613.1.1]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.34/Source/WebKit/Configurations/Version.xcconfig
+[612.1.6]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.32/Source/WebKit/Configurations/Version.xcconfig
```

Comment:
holy shit i messed up that formatting big time. github really needs monospace fonts in the suggestion dialog lol.
And i forgot the link, adding that now too:
```suggestion
| Distro                                                            | webkitgtk | WebKit    | Safari Equivalent |
| ----------------------------------------------------------------- | --------- | --------- | ----------------- |
| Ubuntu 18.04                                                      | 2.20      | [606.1.4] | 12.0              |
| Debian 9 (with backport), Debian 10                               | 2.24      | [608.1.6] | 13.0              |
| Ubuntu 20.04                                                      | 2.28      | [610.1.1] | 14.0              |
| Debian 11, Ubuntu 18.04 (with update), centos 8 (non-stream)      | 2.32      | [612.1.6] | 15.0              |
| Debian 10 (with update)                                           | 2.34      | [613.1.1] | 15.4              |
| Debian 11 (with update), Ubuntu 20.04 (with update), Ubuntu 22.04 | 2.36      | [614.1.6] | TP 140 (16.0)     |

[605.1.3]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.18/Source/WebKit/Configurations/Version.xcconfig
[606.1.4]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.20/Source/WebKit/Configurations/Version.xcconfig
[608.1.6]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.24/Source/WebKit/Configurations/Version.xcconfig
[610.1.1]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.28/Source/WebKit/Configurations/Version.xcconfig
[612.1.6]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.32/Source/WebKit/Configurations/Version.xcconfig
[613.1.1]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.34/Source/WebKit/Configurations/Version.xcconfig
[614.1.6]: https://trac.webkit.org/browser/webkit/releases/WebKitGTK/webkit-2.36/Source/WebKit/Configurations/Version.xcconfig
```